src/app.js

```diff
@@ -155,14 +118,8 @@ export default function (app) {
   }
 
   function setElementData(element, name, value, oldValue) {
-    name = name.toLowerCase()
-
     if (name === "key") {
```

Comment:
since you are comparing strings here, can a switch/case be used here (more performant)

```javascript
switch(name.toLowerCase()) {
  // case stuff here
}
```